### PR TITLE
Add information about PL Inc

### PR DESCRIPTION
**Pull request by nwalters512**
This should help explain the difference between PrairieLearn (the software) and PrairieLearn, Inc. (the company that provides paid hosting and support).

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -233,9 +233,39 @@ export default function Home() {
...
+        <div className="container-md">
+          <Row>
+            <Column>
+              <h2 className="text-white">Open-source. Forever.</h2>
+              <p className="text-white ">
+                PrairieLearn began life as open-source software, and we&apos;re
+                committed to making sure it stays that way. With an active
+                developer community of professors, course staff, and students,
+                PrairieLearn gets better all the time.
+              </p>
```
Comment (nwalters512) [inline src/pages/index.tsx:246]: I moved this back to the homepage. I think this is a pretty core part of PL, and I don't want to bury this in the "about" page.

File: src/pages/about/index.tsx [hunk 2]
```diff
@@ -78,118 +78,116 @@ export default function About() {
...
+              PrairieLearn has to offer without having to become experts in
+              server administration.
+            </p>
+
+            <p>
+              Paid subscriptions cover the cost of running PrairieLearn, and
+              they also fund our developers that operate the service and are
+              responsible for ongoing maintenance, upgrades, code reviews, an
+              support.
+            </p>
```
Comment (nwalters512) [inline src/pages/about/index.tsx:149]: We can probably iterate on this text a bit.

File: src/pages/about/index.tsx [hunk 3]
```diff
@@ -78,118 +78,116 @@ export default function About() {
...
+              option. This ensures that everyone can take advantage of all that
+              PrairieLearn has to offer without having to become experts in
+              server administration.
+            </p>
+
+            <p>
+              Paid subscriptions cover the cost of running PrairieLearn, and
+              they also fund our developers that operate the service and are
+              responsible for ongoing maintenance, upgrades, code reviews, an
+              support.
```
Comment (mwest1066) [inline src/pages/about/index.tsx:148]:
```suggestion
              Paid subscriptions cover the costs of running PrairieLearn servers, and
              they also fund our system administrators who operate the service and are
              responsible for ongoing maintenance, security, upgrades, and support.
```

File: src/pages/about/index.tsx (hunk 2 above)
Comment (mwest1066) [reply src/pages/about/index.tsx:149]: I added some suggested edits, but feel free to take what you want from that.

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**